test/unit/shimmer.test.js

```diff
@@ -51,6 +52,9 @@ describe('shimmer', function() {
             instrumentedModule = module
             ++counter
             onRequireArgs = arguments
+            if (throwsError) {
+              throw new Error('This threw an error! Oh no!')
+            }
           },
```

Comment:
There is an existing test that verifies the module is only loaded once, which should behave the same if there is no error.  If you remove the implementation, this test will fail. I'm not quite sure what you mean by `but does not appear to directly test the new functionality.` -- Do you mean it's not directly testing that the variable is set? This is testing that the instrumentation only occurs once via existing tests. 

https://github.com/newrelic/node-newrelic/pull/450/files#diff-4f9be5e4eb3107fbdf0d69a962cbd634R89 -- This is the test case that, by luck, is already able to test the new functionality.

Reply:
> Do you mean it's not directly testing that the variable is set?

_nod_ yup -- that's what I meant. It can be a little tedious at times, but in addition to some basic "does this thing even work" tests, we _try_ to test that any new state we're setting gets set/unset.  Since you've gone through your tests you probably see sometimes we're better at this than others ;)